### PR TITLE
Clean up tasks

### PR DESCRIPTION
**Pull request by rshriram**
cc @frankbu

### CODE REVIEW

File: _docs/tasks/traffic-management/egress.md [hunk 1]
```diff
@@ -249,8 +249,11 @@ cloud provider specific knowledge and configuration.
    ```
 
 
-## What's next
+## Further reading
 
 * Read more about [egress rules]({{home}}/docs/concepts/traffic-management/rules-configuration.html#egress-rules).
 
-* Learn how to use Istio's [request routing](./request-routing.html) features.
+* Learn how to use setup
```
Comment (frankbu) [inline _docs/tasks/traffic-management/egress.md]: /use /d

File: _docs/tasks/traffic-management/egress.md [hunk 2]
```diff
@@ -249,8 +249,11 @@ cloud provider specific knowledge and configuration.
...
+  [timeouts]({{home}}/docs/reference/config/traffic-rules/routing-rules.html#httptimeout),
+  [retries]({{home}}/docs/reference/config/traffic-rules/routing-rules.html#httpretry),
```
Comment (gyliu513) [inline _docs/tasks/traffic-management/egress.md]: kill the `,` at the end

Comment (rshriram) [reply _docs/tasks/traffic-management/egress.md]: Yeah, thats a long argument.. Some say to use the comma before and. @Rachael-Graham whats the English's take on this? [pun intended :P]

**Comment by rshriram** [comment]
All local tests pass...

File: _docs/tasks/traffic-management/egress.md (hunk 2 above)
Comment (Rachael-Graham) [reply _docs/tasks/traffic-management/egress.md]: Haha--I know that at least for IBM style you use the oxford comma. So "... timeouts, retries, and circuit breakers ..."

File: Gemfile.lock [hunk 3]
```diff
@@ -6,59 +6,59 @@ GEM
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    addressable (2.5.1)
```
Comment (ldemailly) [inline Gemfile.lock]: how come this file changed ?

**Pull request merged**